components/github/github.app.js

```diff
@@ -16,6 +16,28 @@ module.exports = {
         return repos.map(repo => repo.full_name)
       },
```

Comment:
Did you consider replacing a lot of this with some github sdk?

Reply:
we can look at migrating to octokit in the future